tv-show/src/components/features/shows/ShowContainer/ShowContainer.tsx

```diff
@@ -0,0 +1,14 @@
+import { Container } from "@chakra-ui/react";
+import ShowDetailsContainer from "../ShowDetailsContainer/ShowDetailsContainer";
+import ShowReviewSection from "../ShowReviewSection/ShowReviewSection";
+import { useState } from "react";
+
+export default function ShowContainer() {
+  const [averageRating, setAverageRating] = useState<number | undefined>();
+  return (
+    <Container>
+      <ShowDetailsContainer averageRating={averageRating}/>
+      <ShowReviewSection setAverageRating={(avg) => setAverageRating(avg)} />
```

Comment:
This is up for discussion, but you could have just passed `setAverageRating` here. That is all you're doing after all.